build.xml

```diff
@@ -153,7 +153,7 @@ under the License.
     <property name="ooxml.testokfile" location="poi-ooxml/build/testokfile.txt"/>
 
     <property name="ooxml.lite.agent" location="build/dist/maven/poi-ooxml-lite-agent/poi-ooxml-lite-agent-${version.id}.jar"/>
-    <property name="ooxml.lite.report" location="build/ooxml-lite-report"/>
+    <property name="ooxml.lite.report" location="src/resources/ooxml-lite-report"/>
```

Comment:
FYI, I did not change or check if Ant-build still does this correctly, we should really get rid of the Ant build now that we did a few Gradle-based releases.